mantidproject/jenkins-scripts
# Create a Jenkins Script For Cancelling All Queued and Running Jobs

Adds a script to the jenkins scripts repo that will cancel all running jobs and clear the queue. This should allow for easier management to avoid having to close each job manually if GitHub throws a fit again and re-runs everything. 

It should also print a list of cancelled and aborted jobs to the output.

To test:
1. Push some changes to a PR or two on the staging server's repo
2. Head to the script console in that server's jenkins controller
3. Run the script in this pr in that window
4. Check the output for the clearing of all queued jobs and all running jobs. 

Fixes mantidproject/mantid#35522
